src/code.sh

```
@@ -74,37 +86,43 @@ _generate_region_vcfs() {
+        temp_vcf="${sample_name}_${chromPos//:/}_${knownRef}_${knownAlt}.tmp.vcf.gz"
 
         bcftools mpileup -d 8000 -f "$reference_fasta_name" "$input_bam_name" \
             -r "$chromPos" -a FORMAT/AD,FORMAT/DP -Ou | \
-            bcftools call -mv -Oz -o "$output_vcf"
+            bcftools call -mv -Oz -o "$temp_vcf"
+        # Use bcftools norm to handle alt/alt cases
+        bcftools norm "$temp_vcf" -f "$reference_fasta_name" -m -any -Oz -o "$output_vcf"
 
-        bcftools view -i "REF=='${knownRef}' && ALT=='${knownAlt}'" -Oz -o "${output_vcf}_filtered.vcf.gz" "$output_vcf"
+        # Apply fitering to output VCF by minimum read depth
+        bcftools view -i "REF=='${knownRef}' && ALT=='${knownAlt}' && FORMAT/DP>${minimum_read_depth}" -Oz -o "${output_vcf}_filtered.vcf.gz" "$output_vcf"
```

Shellcheck (0.10.0)

[warning] minimum_read_depth is referenced but not assigned.

(SC2154)